### ISSUE TITLE
Fix calls to GET healthcheck/status.json are filling action_logs table

### ISSUE THREAD

**Issue by jouve** [passbolt/passbolt_api#487]
## ISSUE NAME

* Passbolt Version: 
  ```
  ./bin/cake passbolt version
  Passbolt CE 4.0.2
  Cakephp 4.4.11
  ```
* Platform and Target: https://github.com/passbolt/passbolt_docker/blob/master/debian/Dockerfile

### What you did
passbolt is deployed using the helm chart for about 1.5 months.

### What happened

action_logs is growing, because of calls to `GET /healthcheck/status.json` from k8s healthchecks.
```
MariaDB [passbolt]> SELECT context, count(*) FROM action_logs GROUP BY 1 ORDER BY 2 DESC LIMIT 10;
+------------------------------+----------+
| context                      | count(*) |
+------------------------------+----------+
| GET /healthcheck/status.json |   746311 |
| GET /resources.json          |     3454 |
| GET /settings.json           |     1630 |
| GET /users.json              |     1603 |
| GET /groups.json             |     1585 |
| POST /resources.json         |     1338 |
| GET /users/me.json           |      995 |
| GET /folders.json            |      896 |
| GET /users/csrf-token.json   |      821 |
| POST /auth/login.json        |      566 |
+------------------------------+----------+
```

So about 96% of entries are `GET /healthcheck/status.json`

### What you expected to happen

I guess /healthcheck/status.json could be skipped from action_logs ?

**Comment by stripthis** [comment]
@jouve thanks for bringing this up, we'll add the /healthcheck/status.json to the excluded list for the logs.

**Comment by ishanvyas22** [comment]
Hey @jouve, this issue has been fixed and released with [v4.2.0](https://community.passbolt.com/t/new-release-v4-2-0/8336).
